rdt/hyper_transformer.py

```diff
@@ -467,13 +469,38 @@ def _sort_output_columns(self):
             output_columns = self.get_final_output_columns(input_column)
             self._output_columns.extend(output_columns)
 
+    def _validate_detect_config_called(self, data):
+        """Warn the user if they fit without calling ``detect_initial_config``.
+
+        Note that this tip will only be printed until the first usage of
+        ``detect_initial_config`` method. After that method has been used once
+        the tip will no longer be shown (for an instance of the HyperTransformer).
+        """
+        if not self._config_detected:
+            warnings.warn(
+                "Tip: You can use the method 'detect_initial_config' to inspect "
```

Comment:
Although the issue specifies to raise this if the detect method wasn't called, I think this warning should show whenever a config isn't present. Instead of checking if the method was called just check if there are any values set in the `field_transformers` and `field_sdtypes`